docs/affinity.md

```diff
@@ -0,0 +1,86 @@
+
+# Affinity
+
+For every service, there is a set of service workers and a set of affine
+Hyperbahn relays (the subset of the relay ring that is responsible for traffic
+to and from the given service).
+The minimum viable Hyperbahn fully connects the affine workers and relays,
+however over time, the relay workers encroach on the maximum file descriptor
```

Comment:
don't say "relay workers", simply say "relays" for clarity... "workers" evokes the service workers